docs/installation/docker/index.rst

```diff
@@ -0,0 +1,195 @@
+Installation w/ Docker
+======================
+
+This guide will step you through setting up your own on-premise Sentry
+in `Docker <https://www.docker.com/>`_.
+
+Dependencies
+------------
+
+* `Docker version 1.10+ <https://www.docker.com/getdocker>`_
+
+Building Container
+------------------
+
+Start by cloning or forking
+`getsentry/onpremise <https://github.com/getsentry/onpremise>`_. This
+will act the base for your own custom Sentry.
+
+Inside of this repository, we have a ``sentry.conf.py`` and
+``config.yml`` ready for :doc:`customizing <../../config>`.
+
+On top of that, we have a ``requirements.txt`` file for
+:doc:`installing plugins <../../plugins>`.
+
+Now we need to build our custom image. We have two ways to do this,
+depending on your environment. If you have a custom registry you're
+going to need to push to::
+
+    REPOSITORY=registry.example.com/sentry make build push
+
+If not, you can just build locally::
+
+    make build
+
+If you plan on running the dependent services in Docker as well, we
+support linking containers.
+
+Running Dependent Services
+--------------------------
+
+*Running the dependent services in Docker is entirely optional*, but
```

Comment:
Your choice here, but bolding this with ``**`` makes it definitely stand out more as opposed to italics.